storage/uri.go

```diff
@@ -17,10 +17,6 @@ import (
 // Declare conformance with fyne.URI interface.
 var _ fyne.URI = &uri{}
 
-// For backwards-compatibility with the now-deprecated ListableURI type, we
-// also declare conformance with that.
-var _ fyne.ListableURI = &uri{}
```

Comment:
Why get rid of this?

Reply:
Because we could not have every URI be listable.
ListableURI was a marker that something was listable, i.e. a directory.
By imposing it on all the tests and code were failing because everything was a directory.

It is now more dynamic